mem.go

```diff
@@ -2,12 +2,57 @@ package sqrl
 
 import (
 	"fmt"
+	"math/big"
 
+	"github.com/dustin/go-humanize"
+	"github.com/likexian/host-stat-go"
 	"github.com/shirou/gopsutil/mem"
 )
 
-func MemInfo() {
-	v, _ := mem.VirtualMemory()
+type Ram struct {
+	total, free, swapSpace uint64
+}
+
+type OS struct {
+	hostname, kernel, version string
+}
 
-	fmt.Println(v)
+func ramOSInfo() (Ram, OS) {
+	v, _ := mem.VirtualMemory()
```

Comment:
why aren't you checking the errors here? https://blog.golang.org/error-handling-and-go